docs/docs/20-usage/10-intro.md

```diff
@@ -1,19 +1,20 @@
-# Your first pipeline
+# "Hello world"
 
-Let's get started and create your first pipeline.
+Let's get started and create a simple pipeline to demonstrate the pipeline syntax and overall behavior.
 
 ## 1. Repository Activation
 
-To activate your repository in Woodpecker navigate to the repository list and `New repository`. You will see a list of repositories from your forge (GitHub, Gitlab, ...) which can be activated with a simple click.
+To enable your repository in Woodpecker, navigate to the repository list and select **New Repository**.
+You’ll be presented with a list of repositories from your forge (e.g., GitHub, GitLab) that can be activated with a simple click.
+Ensure you have administrative rights on the repository in your forge, as Woodpecker needs to add a webhook to detect events like pushes, pull requests, and tags. ￼
 
 ![new repository list](repo-new.png)
 
-To enable a repository in Woodpecker you must have `Admin` rights on that repository, so that Woodpecker can add something
-that is called a webhook (Woodpecker needs it to know about actions like pushes, pull requests, tags, etc.).
+## 2. The first workflow
 
-## 2. Define first workflow
-
-After enabling a repository Woodpecker will listen for changes in your repository. When a change is detected, Woodpecker will check for a pipeline configuration. So let's create a file at `.woodpecker/my-first-workflow.yaml` inside your repository:
+Once your repository is enabled, Woodpecker starts monitoring it for changes.
+When a change is detected, it looks for a pipeline configuration file.
+To set up your first workflow, create a file named `.woodpecker/my-first-workflow.yaml` (feel free to change the file name to your liking) in your repository:
```

Comment:
I think it is better to use single workflow syntax here and, probably, call it pipeline.